### ISSUE TITLE
Allow example doctests

### ISSUE THREAD

**Issue by mforets** [JuliaReach/LazySets.jl#60]
as suggested [here](https://github.com/JuliaReach/LazySets.jl/issues/14#issuecomment-348289126), we can turn our examples into proper doctests.

with [Documenter.jl](https://juliadocs.github.io/Documenter.jl/), this boils down to using [jldoctest](https://juliadocs.github.io/Documenter.jl/stable/man/doctests/#Doctests-1) in triple backticks for each example. i don't know if we need to have the line `using LazySets` in each code block.

**Comment by schillic** [comment]
I found the solution to the `using LazySets` problem [here](https://discourse.julialang.org/t/doctest-error/6366/6):

Add the following in triple-backticks to the doc files (e.g., `representations.md`):
```
@meta
CurrentModule = LazySets
DocTestSetup = quote
    using LazySets
end
```

Several examples still fail because they are broken into parts with text in between, but `Documenter.jl` evaluates them in isolation. For this we should use [labels](https://juliadocs.github.io/Documenter.jl/stable/man/doctests/#Preserving-definitions-between-blocks-1).

I will add the changes to #51.

**Comment by schillic** [comment]
Done. Some remarks:

1. If you use several lines in [REPL mode](https://juliadocs.github.io/Documenter.jl/stable/man/doctests/#REPL-Examples-1) (`julia> ...`), you will have to include empty lines between the commands. Otherwise the variables from the previous commands are not available below.

2. I added a commented line to `make.jl` for skipping the doctests. The motivation is that they take really long (), mainly because they include `Plots.jl`.

3. I changed [an output for Zonotopes](https://github.com/JuliaReach/LazySets.jl/pull/51/commits/ea2b7de0e084792df8b1367ba32075649459235c#diff-0bd68f0d56b9f04f182390e5606ca7d8L69). @mforets: Was this a bug?

From my side this issue can be closed when merging #51.
